README.md

```diff
@@ -26,24 +26,20 @@ You should be able to get them from your favorite package manager.
   1. Clone the latest LLVM code (or use `git pull` if you cloned it before)
```

Comment:
Can you please mention that it should be cloned _into the root of this repository_.